go.mod

```diff
@@ -12,7 +12,6 @@ require (
 	github.com/golang/protobuf v1.5.4
 	github.com/google/uuid v1.6.0
 	github.com/hashicorp/go-version v1.6.0
-	github.com/kubernetes-csi/csi-lib-utils v0.17.0
```

Comment:
@adikul30 You recently changed some code for leader election, after that change csi-lib-utils is not getting used. Currently there is no other user of this library in CSI driver. Please review go.mod and go.sum file changes.